extension/JuridicalPersion.StructureDefinition.json

```diff
@@ -0,0 +1,53 @@
+{
+  "resourceType": "StructureDefinition",
+  "url": "http://bbmri.de/fhir/StructureDefinition/JuridicalPersion",
+  "name": "JuridicalPersion",
+  "status": "draft",
+  "fhirVersion": "4.0.0",
+  "kind": "complex-type",
+  "abstract": false,
+  "context": [
+    {
+      "type": "element",
+      "expression": "Organization"
+    }
+  ],
+  "type": "Extension",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Extension",
+  "derivation": "constraint",
+  "differential": {
+    "element": [
+      {
+        "id": "Extension",
+        "path": "Extension",
+        "short": "Optional Extensions Element",
+        "definition": "Optional Extension Element - found in all resources.",
```

Comment:
I think here should be the description of the JuridicalPersion extension and not the generic one.